### PR TITLE
[BIM] Revit bridge

### PR DESCRIPTION
**Pull request by wielinde**
Get and show BCF viewpoints in Revit

### CODE REVIEW

File: frontend/src/app/modules/bcf/bcf-wp-single-view/bcf-wp-single-view.component.ts [hunk 1]
```diff
@@ -6,8 +6,8 @@ import {PathHelperService} from "core-app/modules/common/path-helper/path-helper
 import { NgxGalleryOptions, NgxGalleryImage, NgxGalleryAnimation } from 'ngx-gallery';
 
 
-export type ViewPointOriginal = { id:string, file_name:string };
-export type ViewPoint = { id:string, fileName:string, fullPath:string };
+export type ViewPointOriginal = { uuid:string, snapshot_id:string, snapshot_file_name:string };
+export type ViewPoint = { snapshotId:string, snapshotFileName:string, snapshotFullPath:string };
```
Comment (openprojectci) [inline frontend/src/app/modules/bcf/bcf-wp-single-view/bcf-wp-single-view.component.ts]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/0299a74c5f45c5e892a5f5385173144daf6b868e/frontend/src/app/modules/bcf/bcf-wp-single-view/bcf-wp-single-view.component.ts#L11"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>

File: frontend/src/app/modules/bcf/bcf-wp-single-view/bcf-wp-single-view.component.ts [hunk 2]
```diff
@@ -4,10 +4,14 @@ import {StateService} from "@uirouter/core";
...
+import {HttpClient, HttpHeaders} from "@angular/common/http";
+import {CurrentProjectService} from "core-components/projects/current-project.service";
+import {BcfPathHelperService} from "core-app/modules/bcf/helper/bcf-path-helper.service";
+import {RevitBridgeService} from "core-app/modules/bcf/services/revit-bridge.service";
 
 
-export type ViewPointOriginal = { id:string, file_name:string };
-export type ViewPoint = { id:string, fileName:string, fullPath:string };
+export type ViewPointOriginal = { uuid:string, snapshot_id:string, snapshot_file_name:string };
+export type ViewPoint = { snapshotId:string, snapshotFileName:string, snapshotFullPath:string };
```
Comment (openprojectci) [inline frontend/src/app/modules/bcf/bcf-wp-single-view/bcf-wp-single-view.component.ts:15]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/98c55f584284951e8068eb3ae591271dd16b20b7/frontend/src/app/modules/bcf/bcf-wp-single-view/bcf-wp-single-view.component.ts#L15"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>

**Comment by openprojectci** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/opf/openproject/blob/98c55f584284951e8068eb3ae591271dd16b20b7/frontend/src/app/modules/ifc_models/ifc-viewer.component.ts#L36">frontend/src/app/modules/ifc_models/ifc-viewer.component.ts#L36</a> - Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by oliverguenther** [comment]
Superseded by https://github.com/opf/openproject/pull/8115

**Pull request closed without merging**